sdk/cdn/Azure.ResourceManager.Cdn/CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (2024-04-23)
 
 ### Features Added
 
-
 - Enable the new model serialization by using the System.ClientModel, refer this [document](https://aka.ms/azsdk/net/mrw) for more details.
+- Update api version to 2024-02-01
```

Comment:
what new features have been added in the new API version? Could you add more details?